File: CoreJava/accountpolymorphism-app/src/com/techlabs/account/SavingAccount.java
package com.techlabs.account;

public class SavingAccount extends Account {

	public SavingAccount(int accno, String name, double amount) {
		super(accno, name, amount);
	}

	@Override
	public void withdraw(double amount) {
		if (this.getBalance() >= 500 && this.getBalance() - amount >= 500)
			this.balance = this.balance - amount;
	}

}

File: DotNET/C#/AccountPolymorphism/AccountPolymorphism/SavingAccount.cs
using System;

namespace AccountPolymorphism
{
    class SavingAccount : Account
    {
        public SavingAccount(int accountnum, int balance, string name) : base(accountnum, balance, name)
        {
        }

        public override void withdraw(int amount)
        {
            if (Balance >= 500 && Balance - amount >= 500)
                Balance = Balance - amount;
            else
                Console.WriteLine("Withdraw failed.Minimum Balance is less");
        }
    }
}
